CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## Unreleased
+
+- fix(android): setContext serializes as Context for Android instead of Extra (#2155)
```

Comment:
`l`: I think these don't need to be uppercase.
```suggestion
- fix(android): setContext serializes as context for Android instead of extra (#2155)
```